### PR TITLE
Added Water Buffalo spawns, Dwarf Mines passage under Mount Sternum

### PR DESCRIPTION
**Pull request by Nohus**

### CODE REVIEW

File: data/markers.json [hunk 1]
```diff
@@ -28621,5 +28621,68 @@
...
+		"x": 33093,
+		"y": 32076,
+		"z": 7
+	},
+	{
+		"description": "Dwarf Mines",
+		"icon": "down",
+		"x": 32427,
+		"y": 31988,
+		"z": 10
```
Comment (mathiasbynens) [inline data/markers.json:28686]: Should this be https://tibiamaps.io/map#32428,31989,10:3?

```suggestion
		"x": 32428,
		"y": 31989,
		"z": 10
```

File: data/markers.json [hunk 2]
```diff
@@ -28621,5 +28621,68 @@
...
+		"y": 32143,
+		"z": 7
+	},
+	{
+		"description": "Water Buffalo spawn",
+		"icon": "star",
+		"x": 33093,
+		"y": 32076,
+		"z": 7
+	},
```
Comment (mathiasbynens) [inline data/markers.json:28680]: Is it useful to have these markers? IIRC you only need 5 bestiary kills and after that you won't need these anymore, no?

**Review by mathiasbynens — CHANGES_REQUESTED** [review]

**Review by mathiasbynens — APPROVED** [review]

File: data/markers.json (hunk 2 above)
Comment (mathiasbynens) [reply data/markers.json:28680]: Nohus explained offline that these are for the mount. Okay, let’s merge them now. We can always move them to the `achievements` pack (or another pack) later.

File: data/markers.json [hunk 3]
```diff
@@ -28621,5 +28621,68 @@
...
+	{
+		"description": "Water buffalo (spawn location)",
+		"icon": "star",
+		"x": 33093,
+		"y": 32076,
+		"z": 7
+	},
+	{
+		"description": "Dwarf Mines",
```
Comment (mathiasbynens) [inline data/markers.json:28682]: This marker was already present at the exact same position, with the label "Dwarfs". :/

Comment (Nohus) [reply data/markers.json:28682]: Whoops! I must have used a map version from before it was added, then added it myselfl

**Pull request merged**